app/services/auto-approval/checks/cost-and-variance-equal-or-less-than-first-time-claim.js

```diff
@@ -0,0 +1,46 @@
+const groupExpensesByType = require('../../notify/helpers/group-expenses-by-type')
+
+const AutoApprovalCheckResult = require('../../domain/auto-approval-check-result')
+
+const CHECK_NAME = 'costAndVarianceEqualOrLessThanFirstTimeClaim'
+const FAILURE_MESSAGE = 'Claim expense costs are outside of the accepted variance from the last manually approved claim'
+const COST_VARIANCE_PERCENTAGE = 0.1
```

Comment:
I've hard-coded it here because there is another story for adding the configuration for auto-approval checks